modules/nodes-containers-dev-fuse-configuring.adoc

```diff
@@ -4,63 +4,136 @@
-= Configuring /dev/fuse on unprivileged pods
+= Configuring /dev/fuse for unprivileged builds in pods
 
-As an alternative to the virtual filesystem, you can configure the `/dev/fuse` device to the `io.kubernetes.cri-o.Devices` annotation to access faster builds within unprivileged pods. Using `/dev/fuse` is secure, efficient, and scalable, and allows unprivileged users to mount an overlay filesystem as if the unprivileged pod was privileged. 
+By exposing the `/dev/fuse` device to an unprivileged pod, you grant it the capability to perform Filesystem in Userspace (FUSE) mounts. This is achieved by adding the `io.kubernetes.cri-o.Devices: "/dev/fuse"` annotation to your pod definition. This setup allows an unprivileged user within the pod to use tools like `podman` with storage drivers such as `fuse-overlayfs` by mimicking privileged build capabilities in a secure and efficient manner without granting full privileged access to the pod.
 
 .Procedure
 
-. Create the pod.
+. Define the pod with `/dev/fuse` access:
 +
-[source,terminal]
+Create a YAML file named `fuse-builder-pod.yaml` with the following content: 
++
+[source,yaml]
 ----
-$ oc exec -ti no-priv -- /bin/bash
+apiVersion: v1
+kind: Pod
+metadata:
+  name: fuse-builder-pod
+  annotations:
+    io.kubernetes.cri-o.Devices: "/dev/fuse" <1>
+spec:
+  containers:
+  - name: build-container
+    image: quay.io/podman/stable <2>
+    command: ["/bin/sh", "-c"]
+    args: ["echo 'Container is running. Use oc exec to get a shell.'; sleep infinity"] <3>
+    securityContext: <4>
+      runAsUser: 1000
 ----
 +
+<1> The `io.kubernetes.cri-o.Devices: "/dev/fuse"` annotation makes the FUSE device available.
+<2> This annotation specifies a container that uses an image that includes `podman` (for example, `quay.io/podman/stable`).
+<3> This command keeps the container running so you can `exec` into it.
+<4> This annotation specifies a `securityContext` that runs the container as an unprivileged user (for example, `runAsUser: 1000`).
+* 
++
+[NOTE]
+====
+Depending on your cluster's Security Context Constraints (SCCs) or other policies, you might need to further adjust the `securityContext` specification, for example, by allowing specific capabilities if `/dev/fuse` alone is not sufficient for `fuse-overlayfs` to operate.
+====
+
+. Create the pod:
++
+Apply the YAML file to create the pod in your OpenShift project:
++
 [source,terminal]
 ----
-$ cat >> Dockerfile <<EOF
-FROM registry.access.redhat.com/ubi9
-EOF
+$ oc apply -f fuse-builder-pod.yaml
 ----
+
+. Verify that the pod is running:
 +
 [source,terminal]
 ----
-$ podman build .
+$ oc get pods fuse-builder-pod
 ----
 
-. Implement `/dev/fuse` by adding the `/dev/fuse` device to the `io.kubernetes.cri-o.Devices` annotation.
+. Access the pod and prepare the build environment:
 +
-[source,yaml]
+Once the `fuse-builder-pod` pod is in the `Running` state, open a shell session into the `build-container` environment:
```

Comment:
Per ISG: [Do not use (only) as a conjunction to mean after or when](https://www.ibm.com/docs/en/ibm-style?topic=word-usage#o)
```suggestion
After the `fuse-builder-pod` pod is in the `Running` state, open a shell session into the `build-container` environment:
```